app/webpacker/components/CompetitionsOverview/CompetitionsFilters.js

```diff
@@ -81,21 +81,23 @@ export function EventSelector({
   onEventSelection,
   eventList = WCA_EVENT_IDS,
   disabled = false,
+  maxEvents = WCA_EVENT_IDS.length,
```

Comment:
Would infinity work better as a default? This default is kind of assuming that the default `eventList` is being used.

Reply:
sure, but WCA_EVENT_IDS.length is practically Infinity